src/Sarif/Autogenerated/Result.cs

```diff
@@ -61,44 +61,42 @@ public SarifNodeKind SarifNodeKind
         // than fail, we automatically set Level to FailureLevel.None. Similarly, if Level is 
         // set to any value other than None we update Kind to ResultKind.Fail
 
-        private ResultKind _kind;
+        private ResultKind? _kind;
 
         /// <summary>
         /// A value that categorizes results by evaluation state.
         /// </summary>
         [DataMember(Name = "kind", IsRequired = false, EmitDefaultValue = false)]
-        [DefaultValue(ResultKind.Fail)]
-        [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
+        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
         [JsonConverter(typeof(Microsoft.CodeAnalysis.Sarif.Readers.EnumConverter))]
-        public ResultKind Kind
+        public ResultKind? Kind
```

Comment:
>Kind [](http://example.com/codeflow?start=27&length=4)

the main change is for Level, I see it is similar, see if we want to change as well